cxl/lib/libcxl.c

```diff
@@ -14541,3 +14541,148 @@ CXL_EXPORT int cxl_memdev_ddr_page_select_get(struct cxl_memdev *memdev)
     cxl_cmd_unref(cmd);
     return rc;
 }
+
+/* DDR REFRESH MODE  SET */
+#define CXL_MEM_COMMAND_ID_DDR_REFRESH_MODE_SET CXL_MEM_COMMAND_ID_RAW
+#define CXL_MEM_COMMAND_ID_DDR_REFRESH_MODE_SET_OPCODE 0xFB32
+#define CXL_MEM_COMMAND_ID_DDR_REFRESH_MODE_SELECT_SET_PAYLOAD_IN_SIZE 4
```

Comment:
payload size can be 1 byte as as per below struct.

Reply:
Updated the size